web/views_avatar.py

```
@@ -138,6 +138,55 @@ def customize_avatar(request):
+            # Save new avatar
+            profile.avatar = avatar_file
+            profile.save()
+
+            # Return the new avatar URL with cache buster
+            import time
+
+            avatar_url = f"{profile.avatar.url}?t={int(time.time())}"
+
+            return JsonResponse({"success": True, "avatar_url": avatar_url})
+        except Exception as e:
+            logger.exception("Error uploading avatar photo: %s", str(e))
```

Ruff (0.15.0)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)